website/content/docs/enterprise/license/faq.mdx

```diff
@@ -0,0 +1,124 @@
+The license changes do not impact customers with the baked-in licensed binaries. In a later release of Consul Enterprise, baked-in binaries will be deprecated.
+
+## Q: What is the product behavior change introduced by the licensing changes?
+
+Starting with Consul Enterprise v1.10.0+ent, a valid license is required on-disk (auto-loading) or as an environment variable for Consul Enterprise to successfully boot-up.
+The in-storage license feature will not be supported starting with Consul Enteprise v1.10.0+ent. All Consul Enterprise clusters using v1.10.0+ent must ensure that there is a valid license on-disk (auto-loaded) or as an environment variable.
+
+## Q: What is the impact on EVAL licenses due to this change?
+
+The 6-hour trial period for EVAL licenses will be deprecated as of Consul Enterprise 1.10.0.  
+This means that any clusters deployed with Consul v1.10.0+ent binaries will need to have a valid license on the disk (auto-loaded) or as an environment variable.
+Failure in providing a valid license key will result in the Consul server agent not starting.
+
+## Q: Is there a grace period when licenses expire?
+
+A license includes an `expiration_date` and a `termination_date`. An enteprise binary will cease to function once the `termination_date` has passed.
+Licenses will now have a 24 hrs grace period. The grace period is the time between license expiry till it terminates.
+As Consul Enterprise approaches the expiration date, warnings will be issued in the system logs.
+
+## Q: Does this affect client agents?
+
+For existing clusters, if the clients agents are using ACLs and have a valid token, then they will be able to retrieve the license from the server.
+If the client agents are not using ACLs, then the client agents will be need to have the license on-disk (auto-loading) or as an environment variable.
+For new Consul clusters using Consul v1.10.0+ent, customers must ensure that there is a valid license on-disk (auto-loaded) or as an environment variable.
+
+## Q: Does this affect snapshot agents?
+
+Same behavior as Consul clients. See answer for [Does this affect snapshot agents? ](faq#q-does-this-affect-client-agents)
+
+## Q: What is the behavior is the license is missing?
+
+Consul server agents will detect the absence of the license and immediately exit.
+
+Consul client agents will attempt to retrieve the license from servers if certain conditions are met: ACLs are enabled, a ACL token is provided to the client agent, the client agents configuration contains `start_join/retry_join` addresses, the start/retry join addresses are addresses of the Consul servers.
+
+Consul snapshot agents will attempt to retrieve the license from servers if certain conditions are met: ACLs are enabled, a ACL token is provided to the client agent, the client agents configuration contains `start_join/retry_join` addresses, the start/retry join addresses are addresses of the Consul servers.
```

Comment:
This is the same set of license retrieval conditions as listed for the Consul clients. Should these two sections be combined into one?